### ISSUE TITLE
Force Respawn

### ISSUE THREAD

**Issue by RuedigerLP** [PGMDev/PGM#867]
it would have a trigger filter that once true would force respawn every player that matches a player filter, if the trigger filter turns false and then true again it would force respawn again, maybe force respawn could also have the options of regular respawns like time, auto, etc.

This would make proper round based gamemodes possible without a lot of hackiness.

**Comment by Pablete1234** [comment]
You can set auto=true in respawn tags, what exactly is the request here?

**Comment by RuedigerLP** [comment]
The idea was to have a function where it was possible to respawn players without killing them. E.g. in kotf after the flag is dropped everyone would be respawned regardless if the players are respawning or alive or what ever everyone would be respawned at the same time to have some thing like a little restart of the match. like the beginning of a new round.

**Comment by Pablete1234** [comment]
The answer to the problem you pose is dynamic portals, which should have a similar effect. @Pugzy was working on dynamic filters & portals as well as kits, that's part of #448 

I think this can be closed

**Comment by lefckntea** [comment]
> The answer to the problem you pose is dynamic portals, which should have a similar effect.

I'm pretty sure you can't quite do what this feature wants with that, especially regarding that force respawns would still have a respawn timer.
The idea is that for round based gamemodes, you wanna have respawning players and alive players enter the round at the same time and it being instantly after the last round seems a bit odd. And you obviously don't wanna kill everyone alive at the end of the round.

Maybe this is possible without force respawns but seems like a logical feature making this easier. I guess it needs dynamic filters anyway though.
tldr ruediger great Suggestion teagod vouch!!!

**Comment by Pugzy** [comment]
I think round-based game modes in PGM would require a little more than the ability to force respawn. It's just not designed for this purpose. Depending on the game-mode the playing area and objectives would be the start of your headache.

As for respawn timers, water drops did the trick in the past and could be used for this purpose.

**Comment by RuedigerLP** [comment]
It would rly only work in conquest gamemodes i guess. One problem with the portals would be that respawning palyers would possibly get delayed if they respawn during everyone getting teleported.

**Comment by Pablete1234** [comment]
You can get anyone in their death-screen to respawn by applying filters on `<respawn>` elements, have one with no-time and auto on a filter that triggers when you're trying to do this reset
